diff --git a/fitsio-sys/ext/cfitsio/CMakeLists.txt b/fitsio-sys/ext/cfitsio/CMakeLists.txt
index 6d6af49..e81dc06 100755
--- a/fitsio-sys/ext/cfitsio/CMakeLists.txt
+++ b/fitsio-sys/ext/cfitsio/CMakeLists.txt
@@ -61,7 +61,7 @@ ENDIF (BUILD_SHARED_LIBS)
 FILE(GLOB H_FILES "*.h")
 
 IF (USE_PTHREADS)
-    FIND_PACKAGE(pthreads REQUIRED)
+    FIND_PACKAGE(Pthreads REQUIRED)
     INCLUDE_DIRECTORIES(${PTHREADS_INCLUDE_DIR})
     ADD_DEFINITIONS(-D_REENTRANT)
 ENDIF()
