diff --git a/fitsio-sys/ext/cfitsio/CMakeLists.txt b/fitsio-sys/ext/cfitsio/CMakeLists.txt
index 6d6af49..6f8a080 100755
--- a/fitsio-sys/ext/cfitsio/CMakeLists.txt
+++ b/fitsio-sys/ext/cfitsio/CMakeLists.txt
@@ -61,7 +61,7 @@ ENDIF (BUILD_SHARED_LIBS)
 FILE(GLOB H_FILES "*.h")
 
 IF (USE_PTHREADS)
-    FIND_PACKAGE(pthreads REQUIRED)
+    FIND_PACKAGE(Pthreads REQUIRED)
     INCLUDE_DIRECTORIES(${PTHREADS_INCLUDE_DIR})
     ADD_DEFINITIONS(-D_REENTRANT)
 ENDIF()
@@ -115,7 +115,11 @@ SET(SRC_FILES
     putcol.c putcolb.c putcold.c putcole.c putcoli.c putcolj.c putcolk.c
     putcoll.c putcols.c putcolsb.c putcolu.c putcolui.c putcoluj.c putcoluk.c
     putkey.c quantize.c region.c ricecomp.c scalnull.c simplerng.c swapproc.c
-    wcssub.c wcsutil.c zlib/zcompress.c zlib/zuncompress.c
+    wcssub.c wcsutil.c
+    zlib/zcompress.c zlib/zuncompress.c
+    zlib/adler32.c zlib/crc32.c zlib/deflate.c zlib/infback.c
+    zlib/inffast.c zlib/inflate.c zlib/inftrees.c zlib/trees.c
+    zlib/uncompr.c zlib/zutil.c
 )
 
 # For future modifications:
@@ -126,15 +130,6 @@ SET(SRC_FILES
 # drvrgsiftp.c is only usable if HAVE_NET_SERVICES & HAVE_GSIFTP are defined:
 #drvrgsiftp.c
 
-# Only include zlib source files if we are building a shared library.
-# Users will need to link their executable with zlib independently.
-IF (BUILD_SHARED_LIBS)
-    set(SRC_FILES ${SRC_FILES}
-        zlib/adler32.c zlib/crc32.c zlib/deflate.c zlib/infback.c
-        zlib/inffast.c zlib/inflate.c zlib/inftrees.c zlib/trees.c
-        zlib/uncompr.c zlib/zutil.c
-        )
-ENDIF()
 
 ADD_LIBRARY(${LIB_NAME} ${LIB_TYPE} ${H_FILES} ${SRC_FILES})
 TARGET_LINK_LIBRARIES(${LIB_NAME} ${PTHREADS_LIBRARY} ${M_LIB})
